BackofficeBundle/Controller/SiteController.php

```diff
@@ -41,6 +43,9 @@ public function formAction(Request $request, $siteId)
         $form->handleRequest($request);
         $message =  $this->get('translator')->trans('open_orchestra_backoffice.form.website.success');
         if ($this->handleForm($form, $message)) {
+            if($site->getAliases()->toArray() !== $aliases) {
+                $this->dispatchEvent(SiteAliasEvents::SITEALIAS_UPDATE, new SiteAliasEvent($site));
```

Comment:
Why do you dispatch a SITEALIAS_UPDATE with an event which contains only the siteId.

I think that you could have bind the route update on the siteUpdate directly